src/allure-nunit-tests/allure-results/environment.xml

```diff
@@ -0,0 +1,14 @@
+<environment>
```

Comment:
why this file is here?

Reply:
I added sample configuration files for this setting so I could see it flow through to the local report. 

Might be a useful reference for anyone looking into producing a fully featured report. 

Happy to amend and remove.

Reply:
Ok, if it shown here - let's keep

![image](https://user-images.githubusercontent.com/6696821/123408996-5adb0780-d5b6-11eb-8369-b9d2e3a2e58e.png)